prime-router/src/main/kotlin/SenderConfiguration.kt

```diff
@@ -0,0 +1,10 @@
+package gov.cdc.prime.router
+
+import com.fasterxml.jackson.annotation.JsonCreator
+
+data class SenderConfiguration
+@JsonCreator constructor(
+    val processingModeCode: String? = null,
+    val testingLabName: String? = null,
+    val testingLabCLIA: String? = null
```

Comment:
Don't need to do this now, but I suggest we refactor to make this generic.   That is instead of 

```
configuration:
  processingModeCode: "T"
  testingLabName: "Mo and Mo Testing"
```

we could do something along the lines of:
```
configuration:
    - action: additional-item
      name: processing_mode_code
      value: "T"
    - action: additional-item
      name: testingLabName
      value: "Mo and Mo Testing"
    - action: force-deidentify    // action is a dumb name, but the idea is you might do other things besides add default values. 
```

Another option would be to look at following the same syntax as the default option on http parameters, eg something like
`   - default:  "processing_mode_code:T"
`
However, the ":" might throw yaml into conniption fits without the quotes?

If we do this here, we should also do it for other places where default values are set in our yaml, so we keep a simple consistent syntax.

Reply:
I did think about making it more generic, and allowing us to add whatever we wanted into the defaults collection, and I do like the open-ended nature of this, but I was also concerned with making it too easy to shoot ourselves in the foot.

Reply:
Agreed.  Let's leave as is - if this turns out to be widely used, then we genericize.